p2p/peer.py

```diff
@@ -670,8 +671,14 @@ def unsubscribe(self, subscriber: PeerPoolSubscriber) -> None:
         if subscriber in self._subscribers:
             self._subscribers.remove(subscriber)
 
+    def start_peer(self, peer):
+        asyncio.ensure_future(peer.run(finished_callback=self._peer_finished))
+        self.add_peer(peer)
+
     def add_peer(self, peer):
+        self.logger.debug('Adding peer ({}) ...'.format(peer))
         self.connected_nodes[peer.remote] = peer
+        self.logger.debug('Number of peers: {}'.format(len(self.connected_nodes)))
```

Comment:
When logging we always use string interpolation -- .format() is only used for exception messages

Reply:
FYI, the reason for that is that logging aggregators can group logging messages if the message is done with `logger.info("message with %s placeholders", thing_for_placeholder)` even when `thing_for_placeholder` is different for each message.

It can't do that aggregation when the string has already been formatted.